scala-cli-src

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+ROOT_DIR="$SCRIPT_DIR"
+(
+   cd "$ROOT_DIR" || exit
+   ./mill cli.standaloneLauncher
+)
+"$ROOT_DIR/out/cli/standaloneLauncher/dest/launcher" "$@"
```

Comment:
We can use `exec` here, which will make the `scala-cli-src` invocation exit with the scala-cli exit code.
```suggestion
exec "$ROOT_DIR/out/cli/standaloneLauncher/dest/launcher" "$@"
```